youtube_transcript_api/test/test_cli.py

```diff
@@ -50,14 +75,26 @@ def test_argument_parsing__languages(self):
         self.assertEqual(parsed_args.json, False)
         self.assertEqual(parsed_args.languages, ['de', 'en'])
 
+    def test_argument_parsing__proxies(self):
+        parsed_args = YouTubeTranscriptCli(
+            'v1 v2 --http-proxy http://user:pass@domain:port'.split()
+        )._parse_args()
+        self.assertEqual(parsed_args.http_proxy, 'http://user:pass@domain:port')
+
+        parsed_args = YouTubeTranscriptCli(
+            'v1 v2 --https-proxy https://user:pass@domain:port'.split()
+        )._parse_args()
+        self.assertEqual(parsed_args.https_proxy, 'https://user:pass@domain:port')
+
     def test_run(self):
         YouTubeTranscriptApi.get_transcripts = MagicMock(return_value=([], []))
         YouTubeTranscriptCli('v1 v2 --languages de en'.split()).run()
 
         YouTubeTranscriptApi.get_transcripts.assert_called_once_with(
             ['v1', 'v2'],
             languages=['de', 'en'],
-            continue_after_error=True
+            continue_after_error=True,
+            proxies={"http": "", "https": ""}
```

Comment:
I guess if no proxy options are set you would want `proxies=None`. Otherwise the `if proxies: ...` calls which happen later on will always be true and therefore `requests.get(..., proxies={...})` will always be called even though no proxy options are set.